docs/src/apimanual.md

```diff
@@ -565,4 +565,39 @@ If `canaddconstraint` returns `false`, then calling `addconstraint!` must throw
 
 ### Package Naming
 
-MOI solver interfaces may be in the same package as the solver itself (either the C wrapper if the solver is accessible through C, or the Julia code if the solver is written in Julia, for example). In some cases it may be more appropriate to host the MOI wrapper in its own package; in this case it is recommended that the MOI wrapper package be named `MathOptInterfaceXXX` where `XXX` is the solver name.
+MOI solver interfaces may be in the same package as the solver itself (either the C wrapper if the solver is accessible through C, or the Julia code if the solver is written in Julia, for example).
+The guideline for naming the file containing the MOI wrapper is `src/MOIWrapper.jl` and `test/MOIWrapper.jl` for the tests.
+In some cases it may be more appropriate to host the MOI wrapper in its own package; in this case it is recommended that the MOI wrapper package be named `MathOptInterfaceXXX` where `XXX` is the solver name.
+
+### Testing guideline
+
+The skeleton below can be used for the wrapper test file of a solver name `FooBar`:
+```julia
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIT = MOI.Test
+const MOIU = MOI.Utilities
+const MOIB = MOI.Bridges
+
+# Include here the functions/sets supported by the solver wrapper (not those that are supported through bridges)
+MOIU.@model FooBarModelData () (EqualTo, GreaterThan, LessThan) (Zeros, Nonnegatives, Nonpositives) () (SingleVariable,) (ScalarAffineFunction,) (VectorOfVariables,) (VectorAffineFunction,)
+
+MOIB.@bridge SplitInterval MOIB.SplitIntervalBridge () (Interval,) () () () (ScalarAffineFunction,) () ()
```

Comment:
Why do users need to see this macro call? Can't MOIB just export a SplitInterval type? The macros calls in this code snippet are enough to scare potential solver authors away. (Only partially kidding.)

Reply:
We could define all bridges in MOIB by default yes, I initially thought that it wasn't necessary and would increase the loading time of MOIB but it could be reconsidered